tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/ct_pid_rid_map_test.py

```diff
@@ -139,7 +139,9 @@ def test_field_cleaning(self):
                 (50003, 345, 500, datetime.fromisoformat('2020-08-17').date(),
                  datetime.fromisoformat('2020-08-17 13:00:00+00:00'), 12),
                 (50004, 789, 800, datetime.fromisoformat('2020-08-17').date(),
-                 datetime.fromisoformat('2020-08-17 12:00:00+00:00'), 13)
+                 datetime.fromisoformat('2020-08-17 12:00:00+00:00'), 13),
+                (50005, 3456, 1000, datetime.fromisoformat('2020-08-17').date(),
+                 datetime.fromisoformat('2020-08-17 11:00:00+00:00'), 14)
```

Comment:
Does this mean that `person_id = 3456` continues to exist in the data after this rule executes?  This means this rule heavily depends on a correct notebook to find any problems.  But a test does exist for a warning log message?

Reply:
@dcampbell-vumc @nishanthpp93 correct me if I'm wrong. We added this in because while it should raise the `WARNING` it should also still remain in the table. This record doesn't have a matching record in the `pid_rid_mapping` table, so it remains the same. However, should there be something in place where if a record doesn't have a match in the pid_rid table it should be removed?

Reply:
@amcgrenera-vumc That is how RT handles unmapped pids.  I think this may be fine for the moment and something to consider.  We do have a CT notebook that checks for pid/rid mappings.  It's just good for this to be a conscious choice instead of just a curious byproduct of the SQL.

Reply:
After talking with @mark-velez , we think the default should be to drop records where the pid/rid can't be mapped.  I think we should merge/close this issue as it is and do this work in a different PR.

Reply:
Ok great, this is good to go then, I will close this out!